src/topology/sheaves/sheaf_condition/unique_gluing.lean

```diff
@@ -43,6 +44,11 @@ namespace presheaf
 
 variables {X : Top.{u}} (F : presheaf (Type u) X) {ι : Type u} (U : ι → opens X)
 
+@[simp] lemma res_π_apply (i : ι) (s : F.obj (op (supr U))) :
+  limit.π (discrete.functor (λ i : ι, F.obj (op (U i)))) i (res F U s) =
+  F.map (opens.le_supr U i).op s :=
+congr_fun (res_π F U i) s
```

Comment:
You should be able to get this lemma automatically now by just putting `@[elementwise]` on `res_π`.